libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/ScheduledMailDialogRenderer/ScheduledMailDialogRenderer.tsx

```diff
@@ -497,7 +498,14 @@ export class ScheduledMailDialogRendererUI extends React.PureComponent<
     };
 
     private renderFiltersMessage = (): React.ReactNode => {
-        const { enableWidgetExportScheduling, hasDefaultFilters } = this.props;
+        const { enableWidgetExportScheduling, hasDefaultFilters, editSchedule } = this.props;
+        if (editSchedule) {
```

Comment:
show an info message that the edited schedule will be saved with the original filters.